internal/gatewayapi/backendtlspolicy.go

```diff
@@ -104,6 +104,22 @@ func (t *Translator) processBackendTLSPolicy(
 					tlsBundle.ALPNProtocols[i] = string(resources.EnvoyProxy.Spec.BackendTLS.ALPNProtocols[i])
 				}
 			}
+			if resources.EnvoyProxy.Spec.BackendTLS != nil && resources.EnvoyProxy.Spec.BackendTLS.ClientCertificateRef != nil {
+				ns := ptr.Deref(resources.EnvoyProxy.Spec.BackendTLS.ClientCertificateRef.Namespace, "")
```

Comment:
```suggestion
				ns := ptr.Deref(resources.EnvoyProxy.Spec.BackendTLS.ClientCertificateRef.Namespace, "")
```
Yes, I’m actually unsure if we should restrict this, it could make sense as its a global certificate that is probably going to be configured by the same persona that is able to provision the envoy proxy resource. wdyt?

Reply:
I suggest to restrict it in the same namespace for now, as the other EG API does.

Reply:
alright I had similar thoughts when writing the comment, i will bring it back and add a restriction.